site/en/docs/extensions/mv3/tut_analytics/index.md

```diff
@@ -28,12 +26,7 @@ have an URL of its own.
 ## Installing the tracking code {: #toc-installing }
 
 The standard Google Analytics tracking code snippet fetches a file named `ga.js` from an SSL
-protected URL if the current page was loaded using the `https://` protocol. **Chrome extensions and
-applications may _only_ use the SSL-protected version of `ga.js`**. Loading `ga.js` over insecure
-HTTP is disallowed by Chrome's default [Content Security Policy][4]. This, plus the fact that Chrome
-extensions are hosted under the `chrome-extension://` schema, requires a slight modification to the
-usual tracking snippet to pull `ga.js` directly from `https://ssl.google-analytics.com/ga.js`
-instead of the default location.
```

Comment:
Does this need to be removed?  It seems like some of it (e.g. the part where it requires a modification to the tracking snippet) might be useful still.

Reply:
I put this back.